libsolidity/formal/SolverInterface.h

```diff
@@ -79,6 +80,18 @@ struct FunctionSort: public Sort
 	}
 };
 
+struct ArraySort: public Sort
+{
+	ArraySort(SortPointer _domain, SortPointer _range):
```

Comment:
Please leave a comment explaining the parameters. I suppose: domain is the type of the values and range is the type of the index?